packages/components/src/components/FileHeaderInfo.js

```diff
@@ -0,0 +1,64 @@
+import { Text } from '@asyncapi/generator-react-sdk';
+
+/**
+ * @typedef {'python' | 'javascript' | 'typescript' | 'java' | 'csharp' | 'rust' | 'dart'} Language
+ * Supported programming languages.
+ */
+
+/**
+ * Mapping of language to comment styles.
+ * @type {Record<Language, string>}
+ */
+const commentStyles = {
+  python: '#',
+  javascript: '//',
+  typescript: '//',
+  java: '//',
+  csharp: '//',
+  rust: '//',
+  dart: '///'
+};
+
+/**
+ * Renders a file header with metadata information such as title, version, protocol, host, and path.
+ *
+ * @param {Object} props - Component props.
+ * @param {object} props.info - Info object from the AsyncAPI document.
+ * @param {object} props.server - Server object from the AsyncAPI document.
+ * @param {Language} props.language - Programming language used for comment formatting.
+ * @returns {JSX.Element} Rendered file header.
+ */
+export function FileHeaderInfo({ info, server, language }) {
+  const commentChar = commentStyles[language] || '//';
+  const line = `${commentChar.repeat(50)}`;
```

Comment:
in case of dart it is a lot, and goes in 2 lines basically

if you want to keep it simple then maybe do 25? 

or maybe we do it wrong and it should look completely different in case of dart - I see they just do `///` in general

it wont be complex if in case of `line` you will also keep a language specific config, like for `commentStyles`